pkg/bgp/speaker/speaker.go

```diff
@@ -277,65 +279,46 @@ func (s *MetalLBSpeaker) OnUpdateEndpointSliceV1Beta1(eps *slim_discover_v1beta1
 	return nil
 }
 
-// OnAddNode notifies the Speaker of a new node.
-func (s *MetalLBSpeaker) OnAddNode(node *v1.Node) error {
+// reconcileNode notifies the speaker of a node (K8s Node or CiliumNode) event
+func (s *MetalLBSpeaker) reconcileNode(op Op, nodeMeta *metav1.ObjectMeta, podCIDRs *[]string, withDraw bool) error {
```

Comment:
nit: 

keep function name consistent with comments. 

```
func (s *MetalLBSpeaker) notifyNodeEvent(op Op, nodeMeta *metav1.ObjectMeta, podCIDRs *[]string, withDraw bool) 
```

Reply:
I like that suggestion. Will change it accordingly!